app/views/event-projection-detail.njk

```diff
@@ -14,7 +14,7 @@
   </div>
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
-      <h1 class="govuk-heading-xl">Event Details</h1>
+      <h1 class="govuk-heading-l">Event Details</h1>
```

Comment:
What's the impact between what it was and what this line was changed to?

Reply:
It changes the headers to the correct size.  xl is too big, we made them by mistake.